ui/src/app/model/issue-report.ts

```diff
@@ -0,0 +1,12 @@
+export default interface IIssueReport {
+    errors: Map<string, IssueDescription>
+    warnings: Map<string, IssueDescription>
+    suggestions: Map<string, IssueDescription>
+    notes: Map<string, IssueDescription>
+}
+
+export interface IssueDescription {
```

Comment:
This interface doesn't contain issue description it only contains table names and table counts, rename appropriately